epitome/functions.py

```diff
@@ -179,6 +179,93 @@ def indices_for_weighted_resample(data, n,  matrix, cellmap, assaymap, weights =
     surrounding = np.unique(list(map(func_, choice)))
     return surrounding[(surrounding > 0) & (surrounding < data_count)].astype(int)
 
+def compute_casv(m1, m2, radii, indices= None):
+    '''
+    Computes CASV between two matrices. CASV indiciates how similar
```

Comment:
https://github.com/YosefLab/epitome/blob/master/epitome/generators.py#L216 see code here to add a continous bool parameter to see whether user wants to include agreement

Reply:
is this for `compute_casv` or for `load_data`?